### ISSUE TITLE
Misleading error message if incorrect directory provided in open file request

### ISSUE THREAD

**Issue by confluence** [CARTAvis/carta-backend#1284]
The GUI file browser ensures that files and directories sent to the backend in open file requests are real, but this restriction can be bypassed through the scripting interface. If an attempt is made to open a file in a non-existent directory, the request fails cleanly with an error message (as expected), but the error message reports that the _file name_ does not exist (which is misleading). The error should refer to the directory instead.

Steps to reproduce:

1. Launch frontend and backend >= 4.0 release (with scripting enabled); install latest Python wrapper; launch Python interpreter
2.
```
from carta.session import Session
session = Session.interact("PUT URL WITH TOKEN HERE", PUT_SESSION_ID_HERE)
img = session.open_image("bad_directory/somefile.fits")
```

Expected behaviour: error dialog reports that `bad_directory` does not exist.

Actual behaviour: error dialog reports that `somefile.fits` does not exist.

**Comment by pford** [comment]
@confluence is the path expected to be absolute, relative to the root folder, or relative to the starting folder?

**Comment by confluence** [comment]
The path sent to the backend is relative to the root folder. The `bad_directory` in my example is relative to the *current* directory, and will be transformed into the full path relative to the root, but the backend doesn't have to care about that -- it's fine if it just prints the full path in the error message (that will also make it clearer to the user how their path was expanded and where they went wrong).

**Comment by confluence** [comment]
Fixed in #1317.
